src/main/resources/static/index.html

```diff
@@ -46,7 +46,7 @@ <h2 class="box-1">Person Record Status Sync</h2>
   </div>
 </div>
 <br>
-<div id="reval-panel" class="hidden">
+<div id="reval-panel" class="">
```

Comment:
The panel is default to be `hidden`, and only when profile doesn't contain nimdta, `hidden` is removed.
Please check the method `fetchSysProfile()` in `main.js`.

Reply:
Whoops, that's just from local testing